app/controllers/crate/index.js

```diff
@@ -130,20 +126,22 @@ export default Ember.ObjectController.extend({
             } else {
                 method = 'delete';
             }
-            var self = this;
-            ajax({ method: method, url: url }).finally(function() {
-                self.set('fetchingFollowing', false);
-            });
+
+            ajax({
+              method,
+              url
```

Comment:
There seems to be somewhat inconsistent introduction of 2-space vs 4-space tabs, could this be sure to use 4-space tabs?